markhor_bringup/launch/teleop.launch

```diff
@@ -2,11 +2,11 @@
     <arg name="joy_config" default="xbox" />
     <arg name="joy_dev" default="/dev/input/js0" />
     <arg name="config_filepath" default="$(find markhor_bringup)/config/$(arg joy_config).config.yaml" />
-    
+
     <node pkg="joy" type="joy_node" name="joy_node">
-      <param name="dev" value="$(arg joy_dev)" />
-      <param name="deadzone" value="0.15" />
-      <param name="autorepeat_rate" value="200" />
+        <param name="dev" value="$(arg joy_dev)" />
+        <param name="deadzone" value="0.15" />
+        <param name="autorepeat_rate" value="200" />
```

Comment:
On serait pas mieux de le switch over au  joy to twist.launch? sa rendrais les deadzone standard across configs

Reply:
Do you mean the `teleop_twist_joy.launch` file ?